quranHome/src/commonMain/kotlin/com/islamversity/quran_home/model/SurahUIModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.islamversity.quran_home.model
+
+data class SurahUIModel(
+
+    val index: Long,
+    val id: String,
+    val order: Long,
+    val name: String,
+    val revealedType: String,
+    val bismillahType: String,
```

Comment:
i think `bismillah` is a flag. and the `revealedType` is a real value in the translation. now that we don't have the types how do we separate them?

Reply:
duplication?!

Reply:
How do we make the difference visible?